src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --roboto-font: 'Roboto', Arial, sans-serif;
+  --header-size: 60px;
+}
+
+html {
+  font-family: var(--roboto-font);
+  padding: 0;
+  margin: 0;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  margin: 0 auto;
+  width: 100%;
```

Comment:
According to the checklist, you should avoid fixing container sizes unless it's required by the task. Here, you've set the width of .header to 100%. If there's no specific requirement for this in the task, it's better to let the content size dictate the container size to avoid overflow or accidental scroll bars.